README.rst

```diff
@@ -146,33 +148,46 @@ Introduction
 
 To get started with matchms, we recommend following our `matchms introduction tutorial <https://blog.esciencecenter.nl/build-your-own-mass-spectrometry-analysis-pipeline-in-python-using-matchms-part-i-d96c718c68ee>`_.
 
-Below is a small example of using matchms to calculate the Cosine score between mass Spectrums in the `tests/testdata/pesticides.mgf <https://github.com/matchms/matchms/blob/master/tests/testdata/pesticides.mgf>`_ file.
+Below is an example of using default filter steps for cleaning spectra, 
+followed by calculating the Cosine score between mass Spectrums in the `tests/testdata/pesticides.mgf <https://github.com/matchms/matchms/blob/master/tests/testdata/pesticides.mgf>`_ file.
 
 .. code-block:: python
 
-    from matchms import Pipeline
-    
-    pipeline = Pipeline()
-    
-    # Read spectrums from a MGF formatted file, for other formats see https://matchms.readthedocs.io/en/latest/api/matchms.importing.html 
-    pipeline.query_files = "tests/testdata/pesticides.mgf"
-    pipeline.prefedined_processing_queries = "basic"
-    pipeline.additional_processing_queries = [
-        ["add_parent_mass"],
-        ["normalize_intensities"],
-        ["select_by_intensity", {"intensity_from": 0.001, "intensity_to": 1.0}],
-        ["select_by_mz", {"mz_from": 0, "mz_to": 1000}],
-        ["require_minimum_number_of_peaks", {"n_required": 5}]
-    ]
-    pipeline.score_computations = [["precursormzmatch",  {"tolerance": 100.0}],
-                                   ["cosinegreedy", {"tolerance": 1.0}],
-                                   ["filter_by_range", {"name": "CosineGreedy_score", "low": 0.2}]]
+    from matchms.Pipeline import Pipeline, create_workflow
```

Comment:
@florian-huber I have updated the readme, to work with the new version. 
Feel free to make any changes. 
The tutorials still point to the older version. Maybe you could update the tutorial before or after the workshops?